server/repository/src/migrations/v2_06_00/add_name_insurance_join.rs

```diff
@@ -0,0 +1,56 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_name_insurance_join"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        let policy_type = if cfg!(feature = "postgres") {
+            "insurance_policy_type"
+        } else {
+            "TEXT"
+        };
+
+        if cfg!(feature = "postgres") {
+            // Postgres changelog variant
+            sql!(
+                connection,
+                r#"
+                    ALTER TYPE changelog_table_name ADD VALUE IF NOT EXISTS 'name_insurance_join';
+                "#
+            )?;
+
+            // Postgres enum variant for policy_type
+            sql!(
+                connection,
+                r#"
+                    CREATE TYPE insurance_policy_type AS ENUM ('Personal', 'Business');
```

Comment:
As discussed, probably needs to be ALL_CAPS like the other PG enums.

Reply:
Done!